scripts/04021-virginiahperiah.py

```diff
@@ -1,4 +1,5 @@
 full_name = 'Virginiah Periah'
 hng_num = '04021'
 used_lang = 'Python'
+email = 'virgyperry@gmail.com'
 print(f" Hello World, this is {full_name} with HNGi7 ID {hng_num} using {used_lang} for stage 2 task.")
```

Comment:
Hi, you only added the email variable but we'd need you to print it after the word `task` and one space. You can find more info [here](https://hngi7.slack.com/archives/C014G2TMQ3G/p1591185461011700)